src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import SearchComponent from './SearchComponent.js';
+import '../styles/Header.css';
+
+const Header = ({searchQuery, onSearchSubmit}) => {
+    return (
```

Comment:
You can simplify the expression in case you have to return only a piece of HTML.
In this case, you can remove the `return` statement and just use the plain arrow function expression `() => <div>`